test/Frontend/embed-bitcode-tvos.ll

```diff
@@ -1,6 +1,6 @@
 ; REQUIRES: CODEGENERATOR=AArch64
 ; RUN: llvm-as %s -o %t.bc
-; RUN: %swift -target arm64-apple-tvos9 -c -module-name someModule -embed-bitcode -disable-llvm-optzns -o %t2.o %t.bc -dump-clang-diagnostics 2> %t.diags.txt
+; RUN: %swiftc_driver_plain -frontend -target arm64-apple-tvos9 -c -module-name someModule -embed-bitcode -disable-llvm-optzns -o %t2.o %t.bc -dump-clang-diagnostics 2> %t.diags.txt
```

Comment:
`%target-swift-frontend` can replace `%swiftc_driver_plain -frontend` here. We'll take the last `-target` on the command line anyways.

Reply:
The reason I went with `%swiftc_driver_plain` is because that avoids the addition of `swift_driver_test_options` or `swift_frontend_test_options`. We are using these configs to specify import paths required for cross compiling, and if the paths get passed to this test it will fail with something like:

```
swift/test/Frontend/embed-bitcode-tvos.ll:16:23: error: CHECK-IMPORTER-NOT: excluded string found in input
; CHECK-IMPORTER-NOT: argument unused
                      ^
<stdin>:2:10: note: found here
warning: argument unused during compilation: '-isystem /some/sysroot'
         ^~~~~~~~~~~~~~~
```

This matches the behavior of the `embed-bitcode.ll` test.